### DISCUSSION TITLE
Delete trashed documents permanently before 30 days

### DISCUSSION THREAD

**Discussion by ghost** [outline/outline#1748] [Ideas]
I've implemented permanent deletion of trashed documents. It works as you would expect; trashed documents receive an extra menu option called "Delete permanently", which reuses code from [`api/utils.gc`](https://github.com/outline/outline/blob/develop/server/api/utils.js) to destroy the document and its attachments.

![CleanShot 2020-12-24 at 16 38 21](https://user-images.githubusercontent.com/76231775/103104586-e6bfa700-4606-11eb-960f-08d357424416.gif)

If you guys want it, I'll submit a PR with the changes.

**Comment by tommoor** [comment]
Please PR, I would accept and appreciate this functionality 👍

**Reply by enzolutions** [reply]
@isemearbot any plans to contribute the feature in a PR?
